cypress/support/cypress-commands/lifecycle.js

```diff
@@ -352,30 +352,20 @@ Cypress.Commands.add('setAppState', (state, appId) => {
           cy.lifecycleSchemaChecks(response, state);
           // TODO: Checks for platform support
         });
-      } else if (
+      } else {
         // Else, send a discovery.launch call to platform to bring app to foreground
-        currentAppState.state != CONSTANTS.LIFECYCLE_STATES.INITIALIZING &&
-        currentAppState.state != CONSTANTS.LIFECYCLE_STATES.FOREGROUND
-      ) {
         cy.launchApp((appType = CONSTANTS.CERTIFICATION), appId);
         appObject.setAppObjectState(state);
-      } else {
-        appObject.setAppObjectState(state);
       }
       break;
 
     // Set state to background
     case CONSTANTS.LIFECYCLE_STATES.BACKGROUND:
       // TODO: Checks for platform support
-      if (
-        currentAppState.state != CONSTANTS.LIFECYCLE_STATES.BACKGROUND &&
-        currentAppState.state != CONSTANTS.LIFECYCLE_STATES.INITIALIZING
-      ) {
-        // If current app state is not background, send message to platform to set app state to background
-        cy.setLifecycleState(state, appId).then(() => {
-          appObject.setAppObjectState(state);
-        });
-      }
+      // Send message to platform to set app state to background
+      cy.setLifecycleState(state, appId).then(() => {
+        appObject.setAppObjectState(state);
+      });
       break;
 
```

Comment:
No need to make any change to background as the api itself is not present in open rpc